cmd/axelard/cmd/vald/tofnd_grpc/client.go

```diff
@@ -0,0 +1,30 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/tendermint/tendermint/libs/log"
+	"google.golang.org/grpc"
+
+	"github.com/axelarnetwork/axelar-core/x/tss/tofnd"
+)
+
+//go:generate moq -pkg mock -out ./mock/rpcClient.go . Client
+
+// Client defines the interface of a grpc client to communicate with tofnd Multisig service
+type Client interface {
+	tofnd.MultisigClient
+}
+
+// Connect connects to tofnd gRPC Server
+func Connect(host string, port string, timeout time.Duration, logger log.Logger) (*grpc.ClientConn, error) {
+	serverAddr := fmt.Sprintf("%s:%s", host, port)
+	logger.Info(fmt.Sprintf("initiate connection to tofnd gRPC server: %s", serverAddr))
+
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	return grpc.DialContext(ctx, serverAddr, grpc.WithInsecure(), grpc.WithBlock())
+}
```

Comment:
is this going to work? Can you safely cancel the context after DialContext returns?